docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -435,6 +435,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+Typically, the rule is implemented by using the demographic parity ratio introduced
+in the :ref:`disparity_metrics` section, with a result considered 'fair' if the ratio
+exceeds 80% for all identified subgroups.
+*Application of this threshold is wrong in many scenarios.*
+
+**What exactly does the rule mean?**
+Legally, the 80% threshold is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
```

Comment:
Maybe add a small sentence to explain what prima facie evidence is.

Reply:
That's what the link is for.

Reply:
In my opinion we should avoid jargon as much as possible, and if we use at least explain it in the text if we do. I'm probably spending too much time on such a detail, but we've seen in other texts (e.g., on construct validity) that using a lot of (new) terminology can make texts less accessible for our primary audience (data scientists). I would definitely count "prima facie" as jargon, but that's perhaps because I'm a non-native English speaker. But it's not a hill I'm willing to die on so if others disagree we can keep it in the current form.